dashboard/src/containers/Dashboard.tsx

```diff
@@ -0,0 +1,276 @@
+                Header: 'Aloitusaika',
+                accessor: 'start',
+            }, {
+                Header: 'Loppumisaika',
+                accessor: 'end',
+            }, {
+                Header: 'Luotu',
+                accessor: 'created',
+            }, {
+                Header: 'Päivitetty',
+                accessor: 'modified',
+            },
+        ];
+
+        return (
+            <main className="main">
+                <Container fluid={true} className="dashboard">
+                    <Row>
+                        <Col xl="7" lg="6" md="6" sm="12">
+                            <Card>
+                                <CardBody>
+                                    <TimeSelect/>
+                                </CardBody>
+                            </Card>
+                            <Card className="parking-histogram">
+                                <CardHeader>Pysäköintimäärät, viimeiset 24 t</CardHeader>
+                                <CardBody>
+                                    <Bar
+                                        data={bar}
+                                        options={{
+                                            maintainAspectRatio: false,
+                                            legend: {display: false},
+                                            scales: {yAxes: [{ticks: {min: 0}}]}
+                                        }}
+                                    />
+                                </CardBody>
+                            </Card>
+                        </Col>
+                        <Col xl="5" lg="6" md="6" sm="12">
+                            <Card>
+                                <CardBody>
+                                    <RegionSelector/>
+                                </CardBody>
+                            </Card>
+                            <Card className="map-card">
+                                {/*<CardHeader>Kartta</CardHeader>*/}
+                                <CardBody>
+                                    <ParkingRegionsMap/>
+                                </CardBody>
+                            </Card>
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col>
+                            <Card>
+                                <CardHeader>Viimeiset pysäköinnit</CardHeader>
+                                <CardBody>
+                                    <ReactTable data={data} columns={columns} />
+                                </CardBody>
+                            </Card>
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col>
+                            <Button
+                                onClick={this.props.onLogout}
+                                color="danger"
+                            >
+                                <i className="fa fa-sign-out"/>{' '}Kirjaudu ulos
```

Comment:
Umm `{' '}`? Is this for styling? If yes, consider CSS.

Reply:
Yeah, same here. Don't know CSS that well...

Reply:
Ok